_inc/jetpack-strings.php

```diff
@@ -1,747 +1,720 @@
-__( "Jetpack's monitor feature", "jetpack" ), // _inc/client/components/jetpack-connect/index.jsx:165
-_x( "Monitor", "Header. Noun: Monitor is a module of Jetpack.", "jetpack" ), // _inc/client/components/jetpack-connect/index.jsx:166
-__( "Be alerted about any unexpected downtime the moment it happens.", "jetpack" ), // _inc/client/components/jetpack-connect/index.jsx:170
-__( "Jetpack's Protect features", "jetpack" ), // _inc/client/components/jetpack-connect/index.jsx:177
-_x( "Protect", "Header. Noun: Protect is a module of Jetpack.", "jetpack" ), // _inc/client/components/jetpack-connect/index.jsx:178
-__( "Guard your site against brute force login attacks, spam, and harmfulmalware injections.", "jetpack" ), // _inc/client/components/jetpack-connect/index.jsx:182
-__( "Backup and restore", "jetpack" ), // _inc/client/components/jetpack-connect/index.jsx:191
-__( "Automatic, real-time backups mean your entire site is always ready to be restored.", "jetpack" ), // _inc/client/components/jetpack-connect/index.jsx:195
-__( "Join the millions of users who rely on Jetpack to enhance and secure their sites. We're passionate about WordPress and here to make your life easier.", "jetpack" ), // _inc/client/components/jetpack-connect/index.jsx:209
-__( "Daily, automated backups (unlimited storage)", "jetpack" ), // _inc/client/components/jetpack-disconnect-dialog/index.jsx:44
-__( "Priority support", "jetpack" ), // _inc/client/components/jetpack-disconnect-dialog/index.jsx:48
-__( "Spam filtering", "jetpack" ), // _inc/client/components/jetpack-disconnect-dialog/index.jsx:52
-__( "Daily, automated backups (unlimited storage)", "jetpack" ), // _inc/client/components/jetpack-disconnect-dialog/index.jsx:60
-__( "Daily, automated malware scanning", "jetpack" ), // _inc/client/components/jetpack-disconnect-dialog/index.jsx:64
-__( "Priority support", "jetpack" ), // _inc/client/components/jetpack-disconnect-dialog/index.jsx:68
-__( "13Gb of high-speed video hosting", "jetpack" ), // _inc/client/components/jetpack-disconnect-dialog/index.jsx:72
-__( "Daily, automated backups (unlimited storage)", "jetpack" ), // _inc/client/components/jetpack-disconnect-dialog/index.jsx:80
-__( "Daily, automated malware scanning with automated resolution", "jetpack" ), // _inc/client/components/jetpack-disconnect-dialog/index.jsx:84
-__( "Priority support", "jetpack" ), // _inc/client/components/jetpack-disconnect-dialog/index.jsx:88
-__( "Unlimited high-speed video hosting", "jetpack" ), // _inc/client/components/jetpack-disconnect-dialog/index.jsx:92
-__( "SEO preview tools", "jetpack" ), // _inc/client/components/jetpack-disconnect-dialog/index.jsx:96
-__( "Site stats, related content, and sharing tools", "jetpack" ), // _inc/client/components/jetpack-disconnect-dialog/index.jsx:103
-__( "Brute force attack protection and downtime monitoring", "jetpack" ), // _inc/client/components/jetpack-disconnect-dialog/index.jsx:107
-__( "Unlimited, high-spee
```

Comment:
That empty value here is a bit odd. It would be nice if we could know where it comes from, like we had with the comments in the old version.

Reply:
Yes, the empty value is a result of the POT file being generated with an empty first entry. I don't think it's going to be a problem.

Reply:
Removed the empty value in one of the latest commits, thanks!